docs/explanation/how_parts_are_built.rst

```diff
@@ -0,0 +1,78 @@
+How parts are built
+-------------------
+
+As described in :ref:`lifecycle`, parts are built in a sequence of steps:
+*pull*, *overlay*, *build*, *stage* and *prime*.
+
+A part is built in a clean environment to ensure that only the base file
+system and its dependencies are present, avoiding contamination from partial
+builds and side effects from other builds. The environment is a file system in
+a container where the root user's home directory is populated with a number of
+subdirectories and configured to use snaps.
+
+.. ### Verify that snap is available in general for non-Snapcraft builds.
+
+Initially, before the *pull* step is run, the ``/root`` directory contains a
```

Comment:
The most common case is to run in `/root` in a VM or container, but `craft-parts` takes a work directory to specify. Not sure whether that's too much detail for here though.

Reply:
change it to _working_ directory
```suggestion
Initially, before the *pull* step is run, the _working_ directory contains a
```